exasol_integration_test_docker_environment/lib/api/common.py

```diff
@@ -181,7 +178,7 @@ def _configure_logging(
         use_job_specific_log_file: bool) -> Iterator[Dict[str, str]]:
     with get_luigi_log_config(log_file_target=log_file_path,
                               log_level=log_level,
-                              use_job_specific_log_file=use_job_specific_log_file) as luigi_config:
+                              use_job_specific_log_file=use_job_specific_log_file) as luigi_config: # type: Path
```

Comment:
why doesn't it recognize this

Reply:
Oh..... surprise surprise: I can remove the explicit type hint now.

I guess I added the type hint before I fixed the return type of `get_luigi_log_config()`